proxy-ssl.rst

```diff
@@ -0,0 +1,8 @@
+ProxySSL Option
```

Comment:
proxyssl is a seperate feature than BinVoucher, please put this into a seperate commit.